tasks/_facts_route_vgw_by_filter.yml

```diff
@@ -9,14 +9,21 @@
 # * aws_vpc_rt_vgw_by_filter
 # * aws_vpc_rt_vgw_propagate_by_filter
 
+- name: combine vgw filter
+  set_fact:
+    _aws_vpc_route_table_vgw_filter: |
+      {{
+        aws_vpc_route_table_vgw_filter_additional + vgw.filter
```

Comment:
for the sake of clarity, could you please explain why there's a need to have an additional filter?

Reply:
Might become more clear once you have a look at the kubernetes PR